README.adoc

```diff
@@ -1425,12 +1425,11 @@ default_url_options[:host] = 'your_site.com'
 === Email Addresses [[email-addresses]]
 
 Format the from and to addresses properly.
-Use the following format:
 
 [source,ruby]
 ----
 # in your mailer class
-default from: 'Your Name <info@your_site.com>'
+default from: email_address_with_name('Your Name', 'info@your_site.com')
 ----
```

Comment:
Good catch! OTOH, I think it's better not to remove it for Rails 6.0 (or lower) users. How about this?

```diff
 Format the from and to addresses properly.
 Use the following format:

 [source,ruby]
 ----
 # in your mailer class
 default from: 'Your Name <info@your_site.com>'
 ----

+If you're using Rails 6.1 or higher, you can use `email_address_with_name` method.
+
+[source,ruby]
+----
+default from: email_address_with_name('Your Name', 'info@your_site.com')
+----
```

Reply:
@koic thanks, updated.